transaction-service/build.gradle.kts

```diff
@@ -105,10 +111,22 @@ kapt {
 
 protobuf {
     protoc {
-        artifact = "com.google.protobuf:protoc:3.19.6"
+        artifact = "com.google.protobuf:protoc:3.24.3"
+    }
+    plugins {
```

Comment:
move proto-related tasks to protobuf module

Reply:
ok